### PR TITLE
Git Strategy and Instructions

### PR DESCRIPTION
**Pull request by csswizardry**
## Description

I’ve added a detailed section to our CONTRIBUTING guidelines that outlines how we expect people to use Git (branching and committing).

## Related Issue
#224

## Motivation and Context
As the project matures and gets more serious, we need to be a lot more diligent with our workflows around Git, etc. This should start to tighten things up.

## How Has This Been Tested?
N/A

## Markup
N/A

## Screenshots
N/A

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Browser Support

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] Opera
- [ ] Safari
- [ ] Mobile Devices
- [ ] Screen-readers (e.g. JAWS, Apple VoiceOver)
- [x] N/A

## Checklist

- [ ] All code includes full inline documentation (as per the [template](https://github.com/sky-uk/toolkit/blob/master/_template.scss)).
- [ ] All code conforms to the Toolkit [coding style](https://github.com/sky-uk/toolkit/wiki/Coding-Style).
- [ ] All code conforms to [WCAG 2.0 level AA Accessibility Guidelines](https://www.w3.org/TR/WCAG20/).
- [ ] New functions and mixins have appropriate tests.
- [ ] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.
- [ ] Package version updated.
- [ ] CHANGELOG.md updated.
- [x] N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <!--
+                                 W A R N I N G
```
Comment (joebell93) [inline .github/PULL_REQUEST_TEMPLATE.md]: image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,13 +29,159 @@ default.
...
+|            *      \     *
+|            |       \    *
+|            |        \   |
+```
+
+`tkt-#####` is branched off `develop` is branched off `master`.
+
+Work in `tkt-#####` is merged into `develop` via a pull request; work in
+`develop` is merged into `master` on the command line before rolling an official
+release (Core Maintainers only).
```
Comment (joebell93) [inline CONTRIBUTING.md]: I think potentially make it bold:
> (**Core Maintainers only**)

Comment (csswizardry) [inline CONTRIBUTING.md]: Done in aa845a7

**Comment by csswizardry** [comment]
Basically, I’m just keen to avoid things like:

```
aa845b3 All the things
93bc1b1 Fix typos
186aac8 More work
ba34c57 Typo
9f0add9 Misc fixes
5210b06 Fixes
```

…from ending up out there in the open.

**Comment by joebell93** [comment]
+ this will **massively** help towards an automatic CHANGELOG process

**Comment by csswizardry** [comment]
🎉

**Pull request merged**